engine/apps/alerts/incident_appearance/renderers/slack_renderer.py

```diff
@@ -301,7 +310,7 @@ def _get_buttons_blocks(self):
             buttons.append(resolution_notes_button)
 
             # Declare incident button
-            if self.alert_group.channel.organization.is_grafana_incident_enabled:
+            if self.alert_group.channel.organization.is_grafana_incident_enabled and not self.alert_group.acknowledged:
```

Comment:
shouldn't you still be able to see the Declare Incident button even if the Alert Group is ack'd?

Reply:
if it is ack'd we add the button in the first place

Reply:
gotcha 👍 

what do you think about defining the `incident_button` variable higher up so that we don't need to duplicate the definition on lines 198 and 314?